source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -24,13 +24,17 @@ void MarkDownBlockParser::ParseBlock(std::stringstream &stream)
     case ']': case ')':
     {
         // add these char as token to code gen list
-        m_parsedResult.AddNewTokenToParsedResult((char)stream.get());
+		char streamChar;
```

Comment:
is it to remove static cast?

Reply:
there are two forms of `get` function relevant here. the first takes no arguments and returns an `unsigned char`. the second can take a `char&` as an out parameter. rather than hard casting from a signed value to an unsigned one, using the latter form puts the responsibility on stl to Do The Right Thing :)

Reply:
cool; thanks

---
In reply to: [207014840](https://github.com/Microsoft/AdaptiveCards/pull/1750#discussion_r207014840) [](ancestors = 207014840)